CHANGELOG.md

```diff
@@ -8,6 +8,13 @@
 #### Fixed
 #### Removed
 
+### [0.7.28](https://github.com/redpanda-data/helm-charts/releases/tag/console-0.7.28) - 2024-08-08
+#### Added
+#### Changed
+#### Fixed
+* Fixed kubeVersion to be able to deploy on AWS EKS clusters.
+#### Removed
+
```

Comment:
Could this section be moved to 
https://github.com/redpanda-data/helm-charts/blob/b35531cf8de4b173024e12b697ba8cce678a7c27/CHANGELOG.md?plain=1#L130
?

I didn't realised this section is for Redpanda chart not console.

Reply:
For sure, sorry didn't see there is different sections in the changelog. Moved it to the console section.